bin/cml.js

```diff
@@ -89,13 +89,30 @@ const handleError = (message, error) => {
           description: 'Maximum log level',
           choices: ['error', 'warn', 'info', 'debug'],
           default: 'info'
+        },
```

Comment:
This is adding the parameters to commands like `tensorboard`

Reply:
Resolved with https://github.com/iterative/cml/pull/1073/commits/3c05aaa76ba053ebbafbd241e1b8c1b8e1ada3dc